src/App.js

```diff
@@ -1,22 +1,19 @@
 import React from 'react';
 import './App.css';
-
+import TodoList from './TodoList';
 import todos from './api/todos';
 import users from './api/users';
 
 function App() {
+  const newTodos = todos.map(todo => ({
+    ...todo,
+    user: users.find(user => user.id === todo.userId),
+  }));
+
   return (
     <div className="App">
-      <h1>Static list of todos</h1>
-      <p>
-        <span>Todos: </span>
-        {todos.length}
-      </p>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
+      <h1 className="header__text">Static list of todos</h1>
+      <TodoList todo={newTodos} />
```

Comment:
```suggestion
      <TodoList todoList={newTodos} />
```
Ты же передаешь туда массив, а не одну todo